pkg/store/filenames/filenames.go

```diff
@@ -47,6 +48,12 @@ const (
 	SocketDir = "sock"
 )
 
+// Filenames used under a volume directory
+
+const (
+	DataDisk = "datadisk"
```

Comment:
Please update `docs/internal.md` to explain the new files and directories
https://github.com/lima-vm/lima/blob/master/docs/internal.md#instance-directory-lima_homeinstance